src/overloads/arrays.jl

```diff
@@ -187,6 +187,22 @@ function LinearAlgebra.logabsdet(A::AbstractMatrix{D}) where {D<:Dual}
     t1, t2 = LinearAlgebra.logabsdet(tracers)
     return (D(p1, t1), D(p2, t2))
 end
+function LinearAlgebra.:\(A::AbstractMatrix{D}, B::AbstractVector) where {D<:Dual}
+    primals, tracers = split_dual_array(A)
+    p = primals \ B
+    t = tracers \ B
+    return D.(p, t)
```

Comment:
These should probably also say `Dual` instead of `D`, as the primal type might be promoted.